class BankError(Exception):
    pass


class HandError(Exception):
    pass
